lib/presentation/crowdactions/crowdaction_form/sections/commitments_section/assign_commitments/commitment_template/commitment_template_widgets/tag_search.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+
+import '../../../../../../../shared/extra/tags_pills.dart';
+import '../../../../../../../theme/constants.dart';
+
+class TagSearchSection extends StatefulWidget {
+  const TagSearchSection({super.key});
+
+  @override
+  State<TagSearchSection> createState() => _TagSearchSectionState();
+}
+
+class _TagSearchSectionState extends State<TagSearchSection> {
+  List<String> tags = ["Days", "Veganuary", "Diet"];
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Padding(
+          padding: const EdgeInsets.all(10.0),
```

Comment:
Search bar divider width isn't equal to the divider above the commitments list. I think setting the horizontal edge insets to 0 should fix it.

Reply:
Is also a little bit darker

Reply:
done